modules/dnn/src/tengine4dnn/src/tengine_graph_convolution.cpp

```diff
@@ -136,22 +135,23 @@ int create_conv_node(graph_t graph, const char* node_name, const char* input_nam
     set_node_attr_int(conv_node, "pad_h1", &pad_h1);
     set_node_attr_int(conv_node, "pad_w1", &pad_w1);
     set_node_attr_int(conv_node, "output_channel", &outch);
+    set_node_attr_int(conv_node, "input_channel", &inch);
     set_node_attr_int(conv_node, "group", &group);
     set_node_attr_int(conv_node, "dilation_h", &dilation_h);
     set_node_attr_int(conv_node, "dilation_w", &dilation_w);
-    set_node_attr_int(conv_node, "activation", &activation);
+  //  set_node_attr_int(conv_node, "activation", &activation);
 
     release_graph_node(conv_node);
 
     return 0;
 }
 
-graph_t create_conv_graph(float *input_data, int inch, int group, int in_h, int in_w,
+graph_t create_conv_graph(const char * layer_name, float *input_data, int inch, int group, int in_h, int in_w,
```

Comment:
functions that you do not expose in tengine_graph_convolution.hpp should be declared as "static"

Reply:
done.